packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -648,13 +648,21 @@ export class RunningSummarizer implements IDisposable {
 						return;
 					}
 
-					// Check for retryDelay that can come from summaryNack or upload summary flow.
-					// Retry the same step only once per retryAfter response.
 					overrideDelaySeconds = result.retryAfterSeconds;
```

Comment:
Geez the way retryAfterSeconds is actually plumbed through it nuts.  Anyway... it works.

Reply:
It makes sense from the point of view of ODSP error. That's where they are used primarily. But we can totally add them to returned object so it's clear.